h-include.js

```diff
@@ -41,35 +41,64 @@ var hinclude;
   hinclude = {
     classprefix: "include_",
 
-    show_content: function (element, req) {
-      var i, include, message, fragment = element.getAttribute('fragment') || 'body';
-      if (req.status === 200 || req.status === 304) {
-        var container = document.implementation.createHTMLDocument().documentElement;
-        container.innerHTML = req.responseText;
+    create_container: function (req) {
+      var container = document.implementation.createHTMLDocument().documentElement;
```

Comment:
This will work on IE10 if it uses `createHTMLDocument(' ')` - the argument is the `title` of the created document.